mono/metadata/assembly.c

```diff
@@ -1008,7 +1012,13 @@ mono_set_rootdir (void)
 
 	/* Solaris 10 style */
 	str = g_strdup_printf ("/proc/%d/path/a.out", getpid ());
+
+#if defined(HAVE_READLINK)
```

Comment:
Whitespace.

Reply:
Readlink is posix. Ifdef should not be needed.
http://pubs.opengroup.org/onlinepubs/9699919799